ch9.md

```diff
@@ -311,14 +311,14 @@ If we want to keep the Observer and Observable separate:
 
 // producer:
 
-var a = Rx.Observable.create( function onObserver(observer){
+var a = Rx.Observable.create( function onObserve(observer){
 	setInterval( function everySecond(){
-		a.onNext( Math.random() );
+		observer.next( Math.random() );
 	}, 1000 );
 } );
 ```
 
-In this snippet, `a` is the Observable, and unsurprisingly, the separate observer is called `observer`; it's able to "observe" some events (like our `setInterval(..)` loop) and publish events to the `a` observable stream.
+In this snippet, `a` is the Observable, and unsurprisingly, the separate observer is called `observer`; it's able to "observe" some events (like our `setInterval(..)` loop) and ~~publish events to the `a` observable stream~~.
```

Comment:
Why are you marking out this phrase? Doesn't the `observer` push (aka "publish") messages into the `a` observable? Isn't that separation of concerns the whole point of distinguishing observer/observable?